crates/pop-cli/src/commands/new/contract.rs

```diff
@@ -75,6 +78,12 @@ impl NewContractCommand {
 
 		is_template_supported(contract_type, &template)?;
 		generate_contract_from_template(name, &path, &template)?;
+
+		// If the contract is part of a workspace, add it to that workspace
```

Comment:
There's a bug I've encountered. To replicate it: Inside the pop-cli folder, if I run `cargo run new contract test`, it doesn't add the contract to the workspace.